inst/workflow/metrics/cou0002.yaml

```diff
@@ -15,7 +15,7 @@ spec:
   Mapped_SeriousAE:
```

Comment:
Should this be `Mapped_AE`, since we are doing the filtering in `Input_Rate()` now?

Reply:
yep!